backend/services/users/src/__init__.py

```diff
@@ -0,0 +1,47 @@
+import sys, os
+from flask import Flask
+
+sys.path.append(os.path.join('..', ''))
+
+from shared import config
+from flask_pymongo import PyMongo
+
+
+app = Flask(__name__)
+
+app.config.from_object(config.Config(''))
+
+mongo = PyMongo(app)
+db = mongo.db
+
+userSchema = {
+    "$jsonSchema": {
+        "bsonType": "object",
+        "required": ["name", "privateKey"],
```

Comment:
private_key instead of privateKey

Reply:
Seems like we won't be needing this at all now